lib/src/value/boolean.ts

```diff
@@ -48,10 +61,25 @@ class SassBooleanInternal extends Value implements SassBoolean {
   toString(): string {
     return this.value ? 'sassTrue' : 'sassFalse';
   }
+
+  // Legacy API support
+
+  static TRUE: SassBooleanInternal;
+  static FALSE: SassBooleanInternal;
```

Comment:
It's unfortunate that legacy fields are polluting this class. Is there any way these could exist solely in the wrapper layer?

Reply:
Not if we want to re-use the non-legacy boolean constants to avoid an extra layer of wrapping. The upside is that this class isn't even visible through the modern API unless someone does something weird like `sassTrue.constructor`, so it's unlikely anyone will actually notice these.

Reply:
> this class isn't even visible through the modern API

Ah, that's what I was missing. Sounds good